app/connectors/PdsAuthCheckerConnector.scala

```diff
@@ -17,46 +17,80 @@
 
 @ImplementedBy(classOf[PdsAuthCheckerConnectorImpl])
 trait PdsAuthCheckerConnector {
-  def check(request: AuthorisationRequest)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[PdsAuthCheckerResponse]
+  def check(request: AuthorisationRequest)(implicit
+      hc: HeaderCarrier,
+      ec: ExecutionContext
+  ): Future[Either[ValidationErrorResponse, PdsAuthCheckerResponse]]
 }
 
 @Singleton
-class PdsAuthCheckerConnectorImpl @Inject() (httpClientV2: HttpClientV2, appConfig: AppConfig) extends PdsAuthCheckerConnector {
-  def check(request: AuthorisationRequest)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[PdsAuthCheckerResponse] = {
+class PdsAuthCheckerConnectorImpl @Inject() (
+    httpClientV2: HttpClientV2,
+    appConfig: AppConfig
+) extends PdsAuthCheckerConnector {
+  def check(request: AuthorisationRequest)(implicit
+      hc: HeaderCarrier,
+      ec: ExecutionContext
+  ): Future[Either[ValidationErrorResponse, PdsAuthCheckerResponse]] = {
 
     val authType = "UKIM"
     val url = appConfig.pdsAuthCheckerUrl.addPathParts("authorisations")
-    val pdsRequest = PdsAuthCheckerRequest(request.date, authType, request.eoris)
+    val pdsRequest =
+      PdsAuthCheckerRequest(request.date, authType, request.eoris)
 
-    httpClientV2.post(url"$url")
+    httpClientV2
+      .post(url"$url")
       .withBody(Json.toJson(pdsRequest))
       .execute[HttpResponse]
-      .flatMap {
-        response =>
-          response.status match {
-            case OK => {
-              response.json
-                .validate[PdsAuthCheckerResponse]
-                .map(
-                  result => Future.successful(result)
-                )
-                .recoverTotal(
-                  error => Future.failed(JsResult.Exception(error))
-                )
-            }
-            case _ => Future.failed(UpstreamErrorResponse(response.body, response.status))
-          }
+      .flatMap { response =>
+        response.status match {
+          case OK =>
+            response.json
+              .validate[PdsAuthCheckerResponse]
+              .map(result => Future.successful(Right(result)))
+              .recoverTotal(error => Future.failed(JsResult.Exception(error)))
+          case _ =>
+            Future.failed(UpstreamErrorResponse(response.body, response.status))
+        }
+      }
+      .recoverWith { case badRequest: BadRequestException =>
+        val responseBody = badRequest.getMessage
+          .split("Response body '")
+          .lastOption
+          .getOrElse("")
+          .stripSuffix("'")
+        try {
+          val responseParsed = Json.parse(responseBody)
+          responseParsed
+            .validate[ValidationErrorResponse]
+            .map(result => Future.successful(Left(result)))
+            .recoverTotal(error => Future.failed(JsResult.Exception(error)))
+        } catch {
+          case e: Exception => Future.failed(e)
+        }
```

Comment:
Validation errors would come back on a case BAD_REQUEST, why are we handling that with a recover block? It's why we're executing with a HttpResponse.